lib/EnzymeCore/src/rules.jl

```diff
@@ -301,8 +305,8 @@ end
 
 
 function is_inactive_from_sig(@nospecialize(interp::Core.Compiler.AbstractInterpreter),
-                              @nospecialize(TT), sv::Core.Compiler.AbsIntState)
-    return isapplicable(inactive, TT; world, method_table, caller)
+                              @nospecialize(TT), sv::Union{Core.Compiler.AbsIntState, Nothing})
```

Comment:
No this doesn't make sense, sv can't be nothing, the other code should drop the edge handling and be used instead

Reply:
This code is morally equivalent to `abstract_applicable`